packages/app-desktop/gui/ConfigScreen/ConfigScreen.tsx

```diff
@@ -80,6 +84,10 @@ class ConfigScreenComponent extends React.Component<any, any> {
 				void this.switchSection(this.props.defaultSection);
 			});
 		}
+		const child = fork(join(__dirname, 'loadFonts'));
+		child.on('message', (fonts: string[]) => {
+			this.setState({ fonts }, () => child.kill());
+		});
```

Comment:
That will fail if the component is unmounted before the loadFonts script returns